src/DGmethods/DGmodel.jl

```diff
@@ -142,23 +155,36 @@ function (dg::DGModel)(dQdt, Q, ::Nothing, t; increment = false)
             hypervisc_indexmap,
             topology.realelems;
             ndrange = ndrange_surface,
-            dependencies = (event,),
+            dependencies = (event, exchange_Q, exchange_auxstate),
```

Comment:
```suggestion
            dependencies = (volumevisc, exchange_Q, exchange_auxstate),
```

Reply:
Makes it easier to understand, ok the dependencies here are the volume computation and the exchange of Q and exchange of aux

Reply:
Okay.  The reason I went with a generic event name was because sometimes kernels are called and sometimes they are not.  Using a generic name let me avoid a bunch of if statements.  Were I don't need a generic name I can do what you suggested.

Reply:
Yeah if it is a `or` condition going with the generic event is totally fine. I just found myself reading through the code and going up and down to check which event we are waiting upon